src/groovy/au/org/emii/gogoduck/job/Job.groovy

```diff
@@ -37,6 +37,10 @@ class Job {
         new URL("$serverUrl/aggr/$uuid")
     }
 
+    URL getStatusUrl() {
+        new URL("$serverUrl/job/$uuid")
```

Comment:
Potentially fix this URL, look at:
https://github.com/aodn/go-go-duck/blob/job_status/grails-app/controllers/au/org/emii/gogoduck/job/JobController.groovy#L22